lib/offline/background.ts

```diff
@@ -0,0 +1,21 @@
+import { jsonApiPost } from "../axios";
+import { processJournal, processTechnical } from "../process";
+import { offlineDB } from "./db";
+
+export async function submitOfflineJournals() {
+  const data = await offlineDB.farmerJournal.toArray();
+  for (const entry of data) {
```

Comment:
Do we want to try to create a bunch of promises and execute them concurrently? Maybe just make this a `TODO`?
```suggestion
  // TODO: make POST requests concurrent
  for (const entry of data) {
```

Reply:
Sure, I can make it runs concurrently with `Promise.all`